### PR TITLE
sharding setup docs

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team not sure if this is the best place for this kind of doc. Open to suggestions if you don't think it is.

### CODE REVIEW

File: corehq/sql_proxy_accessors/README.md [hunk 1]
```diff
@@ -1,20 +1,82 @@
...
+
+2. Migrate the databases
+
+  * SSH into the DB machine
+  
+```
+  $ sudo -iu cchq
+  $ cd www/<env>/current
+  $ source python_env/bin/activate
+  $ ./manage.py migrate_multi --noinput
```
Comment (czue) [inline corehq/sql_proxy_accessors/README.md]: could this be `fab [environment] manage migrate_multi --noinput`?

**Comment by czue** [comment]
seems like an ok place for now. don't have any obviously better thoughts

File: corehq/sql_proxy_accessors/README.md (hunk 1 above)
Comment (snopoke) [inline corehq/sql_proxy_accessors/README.md]: true, will update

Comment (snopoke) [inline corehq/sql_proxy_accessors/README.md]: https://github.com/dimagi/commcare-hq/pull/11264

**Pull request merged**